### DISCUSSION TITLE
@BEDLEVELVISUALIZER is not triggering the pluging when using script run through API like -X POST -d '{ \"command\":\"start\" }'   http://$PRINTER_ADDRESS:$PRINTER_PORT/api/job

### DISCUSSION THREAD

**Discussion by patisa06** [jneilliii/OctoPrint-BedLevelVisualizer#668] [Q&A]
Dear all, we are using a farm and running all scripts through the API using the start post.
We had notice that the bedlevelling is not updated in the config.yaml using this, but if we open the octoprint front-end and run the script containing 
@BEDLEVELVISUALIZER 
G81
the bed is updated. Do you have an idea why the connection beween the API/start and the bedleveling plugin is not activated ?
Thanks a lot for your help
Patrick

**Comment by jneilliii** [comment]
starting a print job has nothing to do with probing/leveling, you would typically put those commands into your slicer's start gcode. another approach is to add the commands to your `before print job starts` **gcode scripts** in OctoPrint's settings. or use the API to send those commands directly.
